template/YYYY-MM-feature-spec-template.md

```diff
@@ -0,0 +1,74 @@
+# Topic: TODO
+
+* **Status**: Pending/Approved
+* **Author**: Your name (@YourGitHubUserName)
+
+## Topic Summary
+<!-- A paragraph or two to summarize the topic area. Just define it in summary form so we all know what it is. -->
+
+### Top level goals
+<!-- At the most basic level, what are we trying to accomplish? -->
+
+### Non-goals (out of scope)
+<!-- What are we explicitly not trying to accomplish? -->
+
+## Customer profile and challenges
```

Comment:
Can you do a find and replace to update all uses of `customer` to `user`? Open source projects don't have customers.

Reply:
I believe @Reshrahim will remove this file from the PR in favor of https://github.com/radius-project/design-notes/pull/55

Reply:
yes